includes/html/pages/routing/vrf.inc.php

```diff
@@ -89,16 +103,9 @@
             }
         }
 
-        echo "<div style='margin: 5px;'><table border=0 cellspacing=0 cellpadding=5 width=100%>";
         $i = '1';
         foreach (dbFetchRows('SELECT `vrf_name`, `mplsVpnVrfRouteDistinguisher`, `mplsVpnVrfDescription` FROM `vrfs` GROUP BY `mplsVpnVrfRouteDistinguisher`, `mplsVpnVrfDescription`,`vrf_name`') as $vrf) {
-            if ($i % 2) {
-                $bg_colour = Config::get('list_colour.even');
-            } else {
-                $bg_colour = Config::get('list_colour.odd');
-            }
-
-            echo "<tr valign=top bgcolor='$bg_colour'>";
+            echo '<tbody><td></td>';
```

Comment:
Is the tbody tag in the wrong place here?  Should it be outside the loop?

Reply:
Page looked good but might have missed something underneath.
ill double check monday, vrf.inc.php gave me the most problems figuring out because of all the loops.